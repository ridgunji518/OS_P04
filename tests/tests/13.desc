Error Test 3: small page allocation does not cross huge_va_offset
