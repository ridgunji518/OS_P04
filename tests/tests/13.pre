./edit-makefile.sh ../solution/Makefile test_13 > ../solution/Makefile.test
cp -f tests/test_13.c ../solution/test_13.c
cd ../solution/
make clean
make -f Makefile.test clean
make -f Makefile.test xv6.img
make -f Makefile.test fs.img
cd ../tests

