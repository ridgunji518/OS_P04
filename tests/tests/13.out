XV6_TEST_OUTPUT allocuvm out of memory
XV6_TEST_OUTPUT malloc fails correctly
